rocketpy/mathutils/function.py

```diff
@@ -32,6 +37,7 @@
     "akima": 2,
     "spline": 3,
     "shepard": 4,
+    "rbf": 5,
```

Comment:
don't you have to also include the `nearest` option here?

Reply:
As explained a bit better in the docs commit [86458aa](https://github.com/RocketPy-Team/RocketPy/pull/691/commits/86458aab47331eaeb59f9c61795f84548c5eb90c), constant and nearest are basically two sides of the same coin, i.e., they are the same (the constant for 1-D is already the nearest in action). Therefore, I preferred to keep the same naming for the extrapolation types, since I believe adding a "nearest" would be redundant.